install.py

```diff
@@ -10,7 +10,19 @@
 import platform
 import warnings
 import datetime
+import importlib
 
+def install_and_import(package):
+    import importlib
+    try:
+        importlib.import_module(package)
+    except ImportError:
+        import pip
+        pip.main(['install', package])
+    finally:
+        globals()[package] = importlib.import_module(package)
+
+install_and_import('distro')
```

Comment:
You can replace all of this with a simple `import distro`, and add that package in bench's `requirements.txt` file

Reply:
@Alchez `install.py` should be able to set up a frappe env with a single command on a fresh server. It should manage all the dependencies it needs. Although they share the same repo, `install.py` isn't a part of bench, so we don't need to update `requirements.txt`. This should work fine.

Reply:
Will this need to be done again in case there's another dependency to be added?